projects/meson/README.md

```diff
@@ -35,19 +35,22 @@ pip3 install meson ninja
 
 ## Build RevBayes
 
-1. Download RevBayes from our github repository. Clone the repository using git by running the following command in the terminal:
+1. Download RevBayes from our github repository. Clone the repository using git by running the following commands in the terminal:
 
     ``` sh
     git clone https://github.com/revbayes/revbayes.git revbayes
+    cd revbayes
+    git checkout development           # Probably you want the development branch
+    git submodule init
+    git submodule update               # Download tests required for the build
     ```
 
 1. Configure and compile Revbayes:
    
     ``` sh
-    ( cd revbayes/ ; git checkout development )  # Probably you want the development branch
-    ( cd revbayes/ ; git submodule init ; git submodule update ) # Download tests required for the build
-    ( cd revbayes/projects/meson/ ; ./generate.sh )
-    meson build revbayes --prefix=$HOME/Applications/revbayes
+    cd revbayes
+    projects/meson/generate.sh
```

Comment:
This line does not work anymore, because local files are required. Error message:
```
./projects/meson/generate.sh: line 7: ./generate_help.sh: No such file or directory
```

Reply:
:-). You are right.

A more general comment: If somebody reads these instructions, they already found `./projects/meson/`. Should we make these instructions accessible from the main readme? Or at least refer to the different ways of installing revbayes from there?